.gitignore

```
@@ -23,5 +23,4 @@ npm-debug.log
 .pa11yci.js
 .stylelintrc
 dist
-docs
-.vscode/
+docs
```

We have newline at the end of our files
```suggestion
docs

```

---

79332f502e06db15e5e2b16430e1943cd03ff785